mailbox/api/src/main/java/org/apache/james/mailbox/acl/PositiveUserACLDiff.java

```diff
@@ -18,53 +18,35 @@
  ****************************************************************/
 package org.apache.james.mailbox.acl;
 
-import java.util.Map;
 import java.util.stream.Stream;
 
 import org.apache.james.mailbox.model.MailboxACL;
 
-public class PositiveUserACLDiff {
+public class PositiveUserACLDiff extends ACLDiff {
```

Comment:
Do that with composition, not inheritance (!)

It means that aclDiff  should be a field of this class.